recipe/meta.yaml

```
@@ -19,31 +19,40 @@ requirements:
   host:
     - python >=3.5
     - pip
+    - hatchling >=1.12.2
+    - hatch-vcs >=0.2.0
+    - wheel
   run:
     - python >=3.5
```

```suggestion
    - python >=3.8
```

According to [upstream](https://github.com/conda/conda-content-trust/blob/ebe40421dbf47a72314ea92d5fd29519652afb21/pyproject.toml#L28)